### PR TITLE
[IMP] accounting: new version of cash discount

### PR DESCRIPTION
**Pull request by jcs-odoo**
task-2834752

### CODE REVIEW

File: content/applications/finance/accounting/receivables/customer_invoices/cash_discounts.rst [hunk 1]
```diff
@@ -1,68 +1,180 @@
...
-Start offering the cash discount
 ================================
+Cash discounts and tax reduction
+================================
+
+**Cash discounts** are reductions in the amount a customer must pay for goods or services offered as
+an incentive for paying their invoice promptly. These discounts are typically a percentage of the
+total invoice amount and are applied if the customer pays within a specified time. The purpose of
+offering cash discounts is to encourage customers to pay their bills quickly, which can help the
+company maintain a steady cash flow.
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/receivables/customer_invoices/cash_discounts.rst:9]:
```suggestion
total invoice amount and are applied if the customer pays within a specified time.
```
It's already explained at the beginning

File: content/applications/finance/accounting/receivables/customer_invoices/cash_discounts.rst [hunk 2]
```diff
@@ -1,68 +1,180 @@
...
+.. _cash-discounts/tax-reductions:
+
+Tax reductions
+--------------
+
+Depending on the country or region, the base amount used to compute the tax can vary, which can lead
+to a **tax reduction**.
+
+To configure how the tax reduction is applied, go to :menuselection:`Accounting --> Configuration
+--> Settings`, and in the :guilabel:`Cash Discount Tax Reduction` feature, select one of the three
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/receivables/customer_invoices/cash_discounts.rst:40]:
```suggestion
--> Settings`, and in the :guilabel:`Cash Discount Tax Reduction` feature, select one of the three
```
Where is that feature ? (I know where it is, and I know I can search for it, but not everybody does)

**Review by toaa-odoo — APPROVED** [review]
Already approving, but have a look at my comments

File: content/applications/finance/accounting/receivables/customer_invoices/cash_discounts.rst (hunk 1 above)
Comment (jcs-odoo) [reply content/applications/finance/accounting/receivables/customer_invoices/cash_discounts.rst:9]: Thanks, I move the interesting part to the end of the first sentence
", which can help the company maintain a steady cash flow."

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**